docker/snaptools/makeCompliantBAM.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+import pysam
+import argparse
+import sys
+import os
+
+def main():
+    parser = argparse.ArgumentParser(description="Make a BAM file with cellular barcodes in the "
+                                                 + "read names compliant by moving them to the CB tag")
+    parser.add_argument('--input-bam', dest="inputbam", help="input bam file")
```

Comment:
```python
    parser.add_argument('--input-bam', help="input bam file", required=True)
    parser.add_argument('--output-bam', help="output bam file", required=True)

    args = parser.parse_args()

    if not os.path.isfile(args.inputbam):
        sys.exit(f"input_bam {args.inputbam} not found.")
```

The "right" approach to validate that a string passed is a file is to subclass the `argparse.Action` object, but I agree that's probably too heavyweight for a script: https://docs.python.org/3/library/argparse.html#action (see the bottom portion of the linked section).